src/java.net.http/share/classes/jdk/internal/net/http/common/DebugLogger.java

```diff
@@ -53,60 +115,65 @@ final class DebugLogger implements Logger {
     static final System.Logger WS = System.getLogger(WS_NAME);
     static final System.Logger HPACK = System.getLogger(HPACK_NAME);
     private static final DebugLogger NO_HTTP_LOGGER =
-            new DebugLogger(HTTP, "HTTP"::toString, Level.OFF, Level.OFF);
+            new DebugLogger(HTTP, "HTTP"::toString, LoggerConfig.OFF);
     private static final DebugLogger NO_WS_LOGGER =
-            new DebugLogger(HTTP, "WS"::toString, Level.OFF, Level.OFF);
+            new DebugLogger(HTTP, "WS"::toString, LoggerConfig.OFF);
     private static final DebugLogger NO_HPACK_LOGGER =
-            new DebugLogger(HTTP, "HPACK"::toString, Level.OFF, Level.OFF);
+            new DebugLogger(HTTP, "HPACK"::toString, LoggerConfig.OFF);
```

Comment:
Not because of this PR, but I think the NO_WS_LOGGER and the NO_HPACK_LOGGER have a typo/bug in the `System.Logger`, they are using. I think the NO_WS_LOGGER should be:
```
new DebugLogger(WS, "WS"::toString, LoggerConfig.OFF);
```
and the NO_HPACK_LOGGER should be:

```
new DebugLogger(HPACK, "HPACK"::toString, LoggerConfig.OFF);
```

Reply:
Good catch. Doesn't really matter since these loggers do nothing though.